vanilla_installer/gtk/widget-partition.ui

```diff
@@ -1,49 +1,236 @@
+            <property name="subtitle" translatable="true">Please select a partition from the options below</property>
+          </object>
+        </child>
+        <child>
+          <object class="AdwExpanderRow" id="abroot_b_part_expand">
+            <child type="prefix">
+              <object class="GtkLabel">
+                <style>
+                  <class name="heading"/>
+                </style>
+                <property name="label">B</property>
+              </object>
+            </child>
+            <property name="title" translatable="true">No partition selected</property>
+            <property name="subtitle" translatable="true">Please select a partition from the options below</property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="root_sizes_differ_error">
+        <style>
+          <class name="error"/>
+        </style>
+        <property name="visible">false</property>
+        <property name="title" translatable="true">Root partitions are not the same size.</property>
+        <property name="description" translatable="true">Please make sure your A and B partitions have the same disk space.</property>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="roots_small_error">
+        <style>
+          <class name="error"/>
+        </style>
+        <property name="visible">false</property>
+        <property name="title" translatable="true">Root partitions are too small.</property>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="home_part">
+        <property name="title" translatable="true">Home Partition</property>
+        <property name="description" translatable="true">The partition to mount under “/home”. This directory stores all user data.</property>
+        <child>
+          <object class="AdwExpanderRow" id="home_part_expand">
+            <property name="title" translatable="true">No partition selected</property>
+            <property name="subtitle" translatable="true">Please select a partition from the options below</property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="home_small_error">
+        <style>
+          <class name="error"/>
+        </style>
+        <property name="visible">false</property>
+        <property name="title" translatable="true">Home partition is too small.</property>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="swap_part">
+        <property name="title" translatable="true">Swap partition:</property>
+        <property name="description" translatable="true">Swap acts as a virtual extension of your RAM and enables hibernation. In order for hibernation to be available, your swap partition must be at least the same size as your RAM.</property>
```

Comment:
```suggestion
        <property name="description" translatable="true">Swap acts as a virtual extension of your RAM and enables hibernation. For hibernation to be available, your swap partition must be at least the same size as your RAM.</property>
```

Reply:
I forgot to commit the new labels for swap 😅. This is replaced by the following:
![image](https://user-images.githubusercontent.com/48367298/219659985-df2a0a6c-f383-4d50-a1c0-790d2f7b1fd1.png)